feat(sso): add jumpcloud SAML (#4710)
getsentry/sentry-docs: src/docs/product/accounts/sso/index.mdx

When prompted with the Rippling Metadata URL, copy this into the Sentry Rippling provider configuration. You will have to complete the Rippling application configuration before completing the sentry provider configuration.

#### Jumpcloud

> In your Jumploud Admin portal, navigate to User Authentication > SSO. Add a new SSO Application, and search for Sentry. Click "Configure". Under the SSO menu, update the SP Entity ID and ACS URLs for your tenant. They should be in the format as described [here](https://docs.sentry.io/product/accounts/sso/saml2/#register-sentry-with-idp). Click "save".

Reviewer: I've formatted this as steps to make it easier for the user to follow and not get lost. 
Also, is the "S" in "save" lowercase in the Jumpcloud UI?  

```suggestion
In your Jumploud Admin portal: 

1. Navigate to **User Authentication > SSO**. 
1. Add a new SSO application and search for Sentry. 
1. Click "Configure". 
1. Under the SSO menu, update the SP Entity ID and ACS URLs for your tenant. They should be in the format as described [here](https://docs.sentry.io/product/accounts/sso/saml2/#register-sentry-with-idp). 
1. Click "save".
```

Author: yes, it is lowercase.

Be sure that you have assigned yourself to the Sentry app in Jumpcloud before attempting these next steps.

> In Sentry, navigate to Settings > Auth in your organization. Click "Configure" on the Jumpcloud provider. Go to the "XML" within the register page. Download your Jumpcloud metadata under the "SSO" tab in your Jumpcloud Sentry SSO app by clicking "Export Metadata". Paste your XML metadata into the text field and click "Parse Metadata".

Reviewer: ```suggestion
1. In [sentry.io](https://sentry.io), navigate to **Settings > Auth** in your organization. 
1. Click "Configure" on the Jumpcloud provider. 
1. Go to the "XML" within the register page. 
1. Download your Jumpcloud metadata under the "SSO" tab in your Jumpcloud Sentry SSO app by clicking "Export Metadata". 
1. Paste your XML metadata into the text field and click "Parse Metadata".
```

> 
> On the "Map Identity Provider" page, fill in 'uniqueID', 'email', 'firstname', and 'lastname' if you have left your Jumpcloud attributes as the defaults for the Sentry app. After this Sentry should be configured for Jumpcloud SSO.

Reviewer: ```suggestion
1. On the "Map Identity Provider" page, fill in 'uniqueID', 'email', 'firstname', and 'lastname' if you have left your Jumpcloud attributes as the defaults for the Sentry app. 

Now, Sentry should be configured for Jumpcloud SSO.
```
